dev/tools/push-images

```diff
@@ -58,12 +59,25 @@ def build_and_push_image_with_docker_buildx(image_name, srcdir, dockerfile_path)
     subprocess.run(build_cmd, cwd=srcdir, check=True)
     print(f"pushed image {image_name}")
 
+def load_kind_image(cluster_name, image_name, srcdir):
+    build_cmd = [
+        "kind",
+        "load",
+        "docker-image",
+        f"--name={cluster_name}",
+        image_name,
+    ]
+
+    subprocess.run(build_cmd, cwd=srcdir, check=True)
+    print(f"loaded image {image_name} into kind cluster {args.kind_cluster_name}")
```

Comment:
Nit: is it cluster_name or args.kind_cluster_name?  I guess this gets to the "options pattern" question.  Not a blocker IMO.